package.json

```diff
@@ -53,7 +53,7 @@
     "steal-stache": "^4.0.0-pre.2",
     "steal-tools": "^1.3.5",
     "through2": "^2.0.1",
-    "uglify-js": "3.3.11"
+    "uglify-js": "3.3.14"
```

Comment:
@matthewp Was there a reason this was pinned? https://github.com/donejs/done-ssr/pull/373